src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lotteryObject = new Lottery();
+        Ball ball1 = lotteryObject.getRandomBall();
```

Comment:
Use for loop for creating several objects of the same class

Reply:
Done!